backend/src/app.ts

```diff
@@ -0,0 +1,63 @@
+import cors from 'cors';
+import express, { Application, Response } from 'express';
+import helmet from 'helmet';
+import morgan from 'morgan';
+import path from 'path';
+import pageNotFoundMiddleware from './middlewares/404.middleware';
+import errorMiddleware from './middlewares/error.middleware';
+import controller from './utils/interfaces/controller.interface';
+
+class App {
+  express: Application;
+
+  port: number;
+
+  constructor(controllers: controller[], port: number) {
+    this.express = express();
+    this.port = port;
+
+    this.initializeMiddlewares();
+    this.intialiseControllers(controllers);
+    this.initializeErrorHandling();
+  }
+
+  private initializeMiddlewares(): void {
+    this.express.use(cors());
+    this.express.use(morgan('dev'));
+    this.express.use(helmet());
+    this.express.use(express.json());
+    this.express.use(express.urlencoded({ extended: false }));
+    this.express.use(express.static(path.join(__dirname, 'public')));
```

Comment:
Why do you need this: `this.express.use(express.static(path.join(__dirname, 'public')));`?